### PR TITLE
Adding Fall24 Meta-release information to CAMARA organization profile README.md

### PR DESCRIPTION
**Pull request by hdamker**
#### What type of PR is this?
* documentation
* project management

#### What this PR does / why we need it:

Adding information about the Meta-release (Picture with APIs and table of the 25 APIs within the Meta-release) within the profile README.md of the CAMARA organization. 

This README.md is part of the Fall24 Meta-release communication - it allows anyone who is coming to the CAMARA GitHub to see on one glance which APIs are available in the Meta-release.

#### Which issue(s) this PR fixes:

n/a

#### Special notes for reviewers

This README.md will show up on the https://github.com/camaraproject home page of CAMARA in GitHub.
The data should be the same as on https://wiki.camaraproject.org/display/CAM/Meta-release+Fall24 
If you see deviation, please comment!

### CODE REVIEW

**Comment by MarkusKuemmerle** [comment]
"Meta-release" is not written in this way at all occurances, please correct

**Comment by hdamker** [comment]
> "Meta-release" is not written in this way at all occurances, please correct

Thanks @MarkusKuemmerle - did some corrections.

It should be "meta-release" if used as word in sentence and "Meta-release" at the beginning of a sentence and within headings.

**Review by MarkusKuemmerle — APPROVED** [review]

**Review by tanjadegroot — APPROVED** [review]

File: profile/README.md [hunk 1]
```diff
@@ -8,17 +8,183 @@ the [Working Groups](https://github.com/search?q=topic%3Aworkinggroup+org%3Acama
 To engage in CAMARA subscribe to our [Mailing Lists](https://lists.camaraproject.org/g/all/subgroups), join the [Community Meetings](https://zoom-lfx.platform.linuxfoundation.org/meetings/telcoapi?view=week) and 
 [contribute here in GitHub](https://github.com/camaraproject/Governance/blob/main/CONTRIBUTING.md) in the repositories with issues, reviews, pull requests or just your questions in the discussions. 
 
-## First Meta-Release of CAMARA APIs coming soon 🚀
+## First Meta-release of CAMARA APIs now available - Fall24 🚀
```
[the remark this answers is not in the document]
Comment (tanjadegroot) [reply profile/README.md:11]: @hdamker Sorry, was stuck in a meeting.

File: profile/README.md [hunk 2]
```diff
@@ -8,17 +8,183 @@ the [Working Groups](https://github.com/search?q=topic%3Aworkinggroup+org%3Acama
...
 [contribute here in GitHub](https://github.com/camaraproject/Governance/blob/main/CONTRIBUTING.md) in the repositories with issues, reviews, pull requests or just your questions in the discussions. 
 
-## First Meta-Release of CAMARA APIs coming soon 🚀
+## First meta-release of CAMARA APIs now available - Fall24 🚀
 
-In the meantime you can check out https://wiki.camaraproject.org/display/CAM/Meta-release+Fall24 for the work in progress on the Meta-Release.
+A CAMARA meta-release combines a set of CAMARA API versions into a consistent release. All API versions in the meta-release fulfill [defined quality criteria](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API-Readiness-Checklist.md) and are compliant to the current CAMARA guidelines in [Commonalities](https://github.com/camaraproject/Commonalities) and [Identity and Consent Management](https://github.com/camaraproject/IdentityAndConsentManagement). There will be two meta-releases per year (March and September).
```
Comment (hdamker) [inline profile/README.md:13]:
```suggestion
A CAMARA meta-release combines a set of CAMARA API versions into a consistent release. All API versions in the meta-release fulfill [defined quality criteria](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API-Readiness-Checklist.md) and are aligned with the current CAMARA guidelines in [Commonalities](https://github.com/camaraproject/Commonalities) and [Identity and Consent Management](https://github.com/camaraproject/IdentityAndConsentManagement). There will be two meta-releases per year (March and September).
```

File: profile/README.md [hunk 3]
```diff
@@ -8,17 +8,183 @@ the [Working Groups](https://github.com/search?q=topic%3Aworkinggroup+org%3Acama
...
 [contribute here in GitHub](https://github.com/camaraproject/Governance/blob/main/CONTRIBUTING.md) in the repositories with issues, reviews, pull requests or just your questions in the discussions. 
 
-## First Meta-Release of CAMARA APIs coming soon 🚀
+## First Meta-release of CAMARA APIs now available - Fall24 🚀
 
-In the meantime you can check out https://wiki.camaraproject.org/display/CAM/Meta-release+Fall24 for the work in progress on the Meta-Release.
+A CAMARA meta-release combines a set of CAMARA API versions into a consistent release. All API versions in the meta-release fulfill [defined quality criteria](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API-Readiness-Checklist.md) and are compliant to the current CAMARA guidelines in [Commonalities](https://github.com/camaraproject/Commonalities) and [Identity and Consent Management](https://github.com/camaraproject/IdentityAndConsentManagement). There will be two meta-releases per year (September and March).
```
[the remark this answers is not in the document]
Comment (hdamker) [reply profile/README.md]: I have chosen "aligned with". Done in https://github.com/camaraproject/.github/pull/3/commits/1b624a32eb41246efd1204ee5e4cfb774fcc2c9a

File: profile/README.md [hunk 4]
```diff
@@ -8,17 +8,183 @@ the [Working Groups](https://github.com/search?q=topic%3Aworkinggroup+org%3Acama
...
+            <td>carrier-billing</td>
+            <td>0.3.0</td>
+            <td><a href="https://github.com/camaraproject/CarrierBillingCheckOut/releases/tag/r1.2">r1.2</a>
+            <td><a href="https://github.com/camaraproject/CarrierBillingCheckOut">CarrierBillingCheckOut</a></td>
+        </tr>
+        <tr>
+            <td>device-reachability-status</td>
+            <td>0.6.0</td>
+            <td><a href="https://github.com/camaraproject/DeviceStatus/releases/tag/r1.2">r1.2</a>
+            <td><a href="https://github.com/camaraproject/DeviceStatusv">DeviceStatus</a></td>
```
Comment (hdamker) [inline profile/README.md:72]:
```suggestion
            <td><a href="https://github.com/camaraproject/DeviceStatus">DeviceStatus</a></td>
```

File: profile/README.md [hunk 5]
```diff
@@ -8,17 +8,183 @@ the [Working Groups](https://github.com/search?q=topic%3Aworkinggroup+org%3Acama
...
+            <td>device-roaming-status-subscriptions</td>
+            <td>0.6.0</td>
+            <td><a href="https://github.com/camaraproject/DeviceStatus/releases/tag/r1.2">r1.2</a>
+            <td><a href="https://github.com/camaraproject/DeviceStatus">DeviceStatus</a></td>
+        </tr>
+        <tr>
+            <td>geofencing-subscriptions</td>
+            <td>0.3.0</td>
+            <td><a href="https://github.com/camaraproject/DeviceLocation/releases/tag/r1.2">r1.2</a>
+            <td><a href="https://github.com/camaraproject/DeviceLocation ">DeviceLocation </a></td>
```
Comment (hdamker) [inline profile/README.md:176]:
```suggestion
            <td><a href="https://github.com/camaraproject/DeviceLocation ">DeviceLocation</a></td>
```

**Review by tanjadegroot — APPROVED** [review]
great work on adding the table ! thanks Herbert

The last sentence says will follow below, but the repo list is not there. you can maybe remove that statement or say "will follow later".

**Comment by hdamker** [comment]
> The last sentence says will follow below, but the repo list is not there. you can maybe remove that statement or say "will follow later".

The README ends up on https://github.com/camaraproject ... and there is the usual listing of repository following.

**Pull request merged**